test/templates/token/data.json

```diff
@@ -3,7 +3,7 @@
     "name": "Token",
     "css_vars": true,
     "rtl": true,
-    "version": "1.0.0",
+    "version": "1.0.5",
```

Comment:
This brings up a good point. Are we trying to still track versions on individual components?

Reply:
we can discuss... we have to figure out the value of version number on components. i thought we could match it to the GD master sketch file, but according to @LeoT7508 there isn't any versioning that is happening right now.